test/js/bun/spawn/spawn.test.ts

```diff
@@ -94,6 +94,7 @@ for (let [gcTick, label] of [
         gcTick();
       });
 
+<<<<<<< Updated upstream
```

Comment:
lol merge conflict committed

Reply:
lol will revert this file, should not be here